concepts/src/concept_csv_export.py

```diff
@@ -24,24 +24,47 @@
 DOCKER = False
 DOCKER_CONTAINER = "openmrs-sdk-mysql"
 VERSION = "2.3"
-LOCALES = ["en"]
-DEFAULT_LOCALE = "en"
+LOCALES = ["en","es"]
+DEFAULT_LOCALE = "es"
 ENCODING = "unicode_escape"
 CONCEPT_KEY_MAPPING: Optional[str] = None
 MAPPING_TYPES: List[str] = [
```

Comment:
This script handles extraction via a SQL query, and as I added in a bunch of other concept sources, we were running up against the max number of joins MySQL supports in a single query.  So, I *excluded* a bunch of mappings and sources from the export, so there may be we few mappings that are missed in the export, however I really don't think these are important.